src/utils/src/kernel_version.rs

```diff
@@ -72,10 +72,23 @@ impl KernelVersion {
     }
 }
 
+pub fn min_kernel_version_for_io_uring() -> KernelVersion {
+    KernelVersion::new(5, 10, 51)
+}
+
+#[macro_export]
+macro_rules! skip_if_io_uring_unsupported {
+    () => {
+        if KernelVersion::get().unwrap() < min_kernel_version_for_io_uring() {
+            return;
+        }
+    };
+}
+
 #[macro_export]
-macro_rules! skip_if_kernel_lt_5_10 {
+macro_rules! skip_if_io_uring_supported {
```

Comment:
I haven't seen a test to check that you are defaulting to the sync version when kernel version is lower than 5.10.

Reply:
Added this to the `test_drive_io_engine `